dev/ProgressRing/ProgressRingIndeterminate.h

```diff
@@ -0,0 +1,32 @@
+﻿#pragma once
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+    // Name:        RadialIntermintent
+    // Frame rate:  60 fps
+    // Frame count: 120
+    // ===========
+    // Property bindings:
+    // Vector4 "Background" as Color
+    // Vector4 "Foreground" as Color
+class ProgressRingIndeterminate :
+    public winrt::implements<ProgressRingIndeterminate, winrt::IAnimatedVisualSource>
+{
+private:
+    winrt::CompositionPropertySet _themeProperties{ nullptr };
+
+    winrt::CompositionPropertySet EnsureThemeProperties(winrt::Compositor compositor);
+
+public:
+    winrt::CompositionPropertySet GetThemeProperties(winrt::Compositor& compositor);
```

Comment:
@simeonc@microsoft.com can we expose the individual properties (Background/Foreground) in this type and map it to the property in the composition property set? That makes it easier to discover and allow us to set it from markup.

Reply:
Sure! The properties we (LottieGen) adds will modify the state of the CompositionPropertySet. The only slight irregularity will be that if you directly modify the CompositionPropertySet then the colors will change but the values that you read out of the properties won't get updated. 
I will prototype something in LottieGen and get your feedback.